### PR TITLE
Add CVE reporting guideline to the repo and readme

### PR DESCRIPTION
**Pull request by JanuszL**
Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

## Category:
**Other** (*e.g. Documentation, Tests, Configuration*)

## Description:
- adds CVE reporting guideline to the repo and readme

### Affected modules and functionalities:
- Sphinx documentation
- GitHub documentation

### Tests:
- NA

- [ ] Existing tests apply
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [x] N/A

### Documentation
- [ ] Existing documentation applies
- [x] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [x] RST
  - [ ] Jupyter
  - [ ] Other
- [ ] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

File: docs/support_matrix.rst [hunk 1]
```diff
@@ -12,12 +12,13 @@ Supported NVIDIA hardware, CUDA, OS, and CUDA driver
...
-  | - Pascal                         |               |                           | - RHEL 7                              |                         |                     |                    |                                                 |
-  | - Maxwell                        |               |                           | - RHEL 8                              |                         |                     |                    |                                                 |
-  | - Kepler                         |               |                           | - and other |PEP599 link|_ compatible |                         |                     |                    |                                                 |
+  | - NVIDIA Hopper GPU architecture | cuda110       | 11.0 and newer,           | - Ubuntu 16.04                        | SM 3.5 and later        | r450 or later       | linux x86_64       | - Prebuild wheel available from developers page |
```
Comment (mzient) [inline docs/support_matrix.rst:15]:
```suggestion
  | - NVIDIA Hopper GPU architecture | cuda110       | 11.0 and newer,           | - Ubuntu 16.04                        | SM 3.5 and later        | r450 or later       | linux x86_64       | - Prebuilt wheel available from developers page |
```

**Review by mzient — APPROVED** [review]
OK, with one nitpick

File: docs/support_matrix.rst (hunk 1 above)
Comment (JanuszL) [reply docs/support_matrix.rst]: Fixed

**Comment by JanuszL** [comment]
!build

**Review by banasraf — APPROVED** [review]

**Comment by szalpal** [comment]
An idea from my side, since we discourage reporting vulnerabilities via github issues, maybe it would be good idea to add such info in the github issue template? That the reporting vulnerability channel is as described in SECURITY.rst?

**Pull request merged**